DependencyInjection/SonataFormatterExtension.php

```diff
@@ -56,25 +56,29 @@ public function load(array $configs, ContainerBuilder $container)
             $loader->load('ckeditor.xml');
         }
 
-        // TODO: To be removed when major version is changed
-        if (!isset($config['default_formatter'])) {
-            reset($config['formatters']);
-            $config['default_formatter'] = key($config['formatters']);
-        }
-
         if (!array_key_exists($config['default_formatter'], $config['formatters'])) {
-            throw new \InvalidArgumentException(sprintf('SonataFormatterBundle - Invalid default formatter : %s, available : %s', $config['default_formatter'], json_encode(array_keys($config['formatters']))));
+            throw new \InvalidArgumentException(sprintf(
```

Comment:
Fabien said it to me, but I am not sure they put it in their CS rules though (and I think the arguments he gave are not good enough, but can understand that he does not want to have to touch 10000 files over this). And anyway, there is a `sprintf` here, and we usually split `sprintf` calls, don't we?